src/app/dashboard/Missions/MissionFundedStatusRow.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Mission } from "../../model";
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import { Box, Container, Grid, Paper } from "@material-ui/core";
```

Comment:
Can we clear these warnings of defining variables but never using them?

Reply:
They're gone!  Do you know if there's a way this stuff can get automatically removed?